test/check-application

```diff
@@ -3179,6 +3179,22 @@ WantedBy=multi-user.target default.target
         self.performContainerAction(IMG_ALPINE, "Start")
         b.wait(lambda: self.getContainerAttr("sys1", "State") == "Running")
 
+    def _testQuadlets(self, *, system: bool = False) -> None:
+        b = self.browser
+        self.login(system=system)
+
+        service_name = "quak"
+        self.createQuadlet(service_name, auth=system)
+        self.waitContainerRow(service_name)
+
+        # Cannot delete a quadlet container, they are managed by .container units
+        b.wait_not_present(self.getContainerAction(service_name, 'Delete'))
```

Comment:
This is always true. getContainerAction only gives you a selector, but you at least need to open the menu toggle and then wait for a positive entry.